docsite/rst/playbooks_loops.rst

```diff
@@ -83,6 +83,9 @@ Suppose you have the following variable::
       bob:
         name: Bob Bananarama
         telephone: 987-654-3210
+        
+    'alice' and 'bob' are the `key` in this example
+    and 'name' and 'telephone' are `value`
```

Comment:
I think this belongs below the second playbook snippet (where we actually use .key and .value) and unindented so that it is rgular body text instead of part of the example playbook code.